sql/commands/sql-show-connections.mdx

```diff
@@ -6,13 +6,14 @@ description: "Use the `SHOW CONNECTIONS` command to see connections that have be
 ## Syntax
 
 ```bash
-SHOW CONNECTIONS [ LIKE_expression ];
+SHOW CONNECTIONS [ FROM schema_name ] [ LIKE_expression ];
 ```
 
 ## Parameters
 
 | Parameter or clause | Description                                                                                                                                                                               |
 | :------------------ | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _schema\_name_      | The schema of the connections to be listed. If not specified, connections from all schemas in the current [schema search path](https://www.postgresql.org/docs/current/ddl-schemas.html#DDL-SCHEMAS-PATH) will be listed. |
```

Comment:
Is it possible to link to the corresponding entry in https://docs.risingwave.com/operate/view-configure-runtime-parameters#how-to-view-runtime-parameters instead? However I didn't find a way to get a URL to the specific table row.